akka-actor-typed-tests/src/test/scala/akka/actor/typed/DeferredSpec.scala

```diff
@@ -26,7 +26,7 @@ object DeferredSpec {
     })
 }
 
-class DeferredSpec extends TypedSpec with StartSupport {
+class DeferredSpec extends TestKit("DeferredSpec") with TypedAkkaSpec {
```

Comment:
Why not use the 0-parameter `TestKit` constructor here?

Reply:
+1, think i added that after converting more tests and getting sick of giving them names